packages/react-liveness/src/components/FaceLivenessDetector/shared/DefaultStartScreenComponents.tsx

```diff
@@ -46,58 +30,45 @@ export const DefaultPhotosensitiveWarning = ({
   return (
     <Flex
       className={`${ComponentClassName.Alert} ${LivenessClassNames.StartScreenWarning}`}
+      style={{ zIndex: '3' }}
     >
       <View flex="1">
-        <View className={ComponentClassName.AlertHeading}>{headingText}</View>
+        {headingText && (
+          <View className={ComponentClassName.AlertHeading}>{headingText}</View>
+        )}
         <View className={ComponentClassName.AlertBody}>{bodyText}</View>
       </View>
       <LivenessIconWithPopover>{infoText}</LivenessIconWithPopover>
     </Flex>
   );
 };
 
-interface DefaultInstructionsProps {
-  headingText: string;
-  goodFitCaptionText: string;
-  goodFitAltText: string;
-  tooFarCaptionText: string;
-  tooFarAltText: string;
-  steps: string[];
+interface DefaultRecordingIconProps {
+  recordingIndicatorText: string;
 }
 
-export const DefaultInstructions = ({
-  headingText,
-  goodFitCaptionText,
-  goodFitAltText,
-  tooFarCaptionText,
-  tooFarAltText,
-  steps,
-}: DefaultInstructionsProps): JSX.Element => {
+export const DefaultRecordingIcon = ({
```

Comment:
We removed the original start screen with instructions and added the recording icon and cancel button as default components that can be overridden